### ISSUE TITLE
Optimizer and Scheduler.

### ISSUE THREAD

**Issue by vishnumh** [RenYurui/Neural-Texture-Extraction-Distribution#21]
Hi Authors,

Thank you for open sourcing the code. The checkpoint doesnt have the final optimizer and scheduler state_dict. Could you please share those ? I  would like to continue training this model with another dataset.
